src/aiida/brokers/rabbitmq/defaults.py

```diff
@@ -29,21 +29,23 @@
 )
 
 
-def detect_rabbitmq_config() -> dict[str, t.Any] | None:
+def detect_rabbitmq_config(**kwargs) -> dict[str, t.Any] | None:
```

Comment:
If we are going to allow arguments, I would prefer they are spelled out since we know exactly which ones are accepted

Reply:
I agree, especially for public functions. Here I was being a bit lazy I guess. ^^